VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -2496,7 +2556,10 @@ public string EthnicityLiteral
                 InputRaceAndEthnicityObs.Component.RemoveAll(c => c.Code.Coding[0].Code == NvssEthnicity.Literal);
                 Observation.ComponentComponent component = new Observation.ComponentComponent();
                 component.Code = new CodeableConcept(CodeSystems.ComponentCode, NvssEthnicity.Literal, NvssEthnicity.Literal, null);
-                component.Value = new FhirString(value);
+                if (!String.IsNullOrEmpty(value))
```

Comment:
For fields like this that create an observation as part of the setter, I think the IsNullOrEmpty check should be wrapped around all of the code in the setter. Check with Saul to see what the best practice is for null values in fhir